### PR TITLE
Added image compression using image_transport_plugin package

### PR DESCRIPTION
**Pull request by TVerloop**
Added image compression using image_transport_plugin package
And additional features for the cameras using rgbd_launcher

### CODE REVIEW

**Comment by TVerloop** [comment]
Just saw someone was quicker than me. If you first handler fmrico's pull request i'll pull that and add image_transport_plugin to that.

**Comment by fmrico** [comment]
@TVerloop Great!!!!

File: build_ros1.sh [hunk 1]
```diff
@@ -51,7 +51,7 @@ docker run -it --rm \
            export PATH=/home/nao/${INSTALL_ROOT}/Python-${PYTHON2_VERSION}/bin:$PATH && \
            cd pepper_ros1_ws && \
            vcs import src < pepper_ros1.repos && \
-           touch src/orocos_kinematics_dynamics/python_orocos_kdl/CATKIN_IGNORE && \
+	       touch src/orocos_kinematics_dynamics/python_orocos_kdl/CATKIN_IGNORE && \
```
Comment (esteve) [inline build_ros1.sh]: Could you revert this to spaces instead of tabs? Thanks!

File: ctc-cmake-toolchain.cmake [hunk 2]
```diff
@@ -235,6 +235,7 @@ set(TIFF_INCLUDE_DIR "${ALDE_CTC_CROSS}/tiff/include" CACHE INTERNAL "" FORCE)
 set(PNG_LIBRARY "${ALDE_CTC_CROSS}/png/lib/libpng.so" CACHE INTERNAL "" FORCE)
 set(PNG_PNG_INCLUDE_DIR "${ALDE_CTC_CROSS}/png/include" CACHE INTERNAL "" FORCE)
 
+
```
Comment (esteve) [inline ctc-cmake-toolchain.cmake]: Please remove this empty line.

**Comment by TVerloop** [comment]
Fixed whitespace in 5a296ef

**Pull request merged**